var/spack/repos/builtin/packages/py-metpy/package.py

```diff
@@ -11,9 +11,12 @@ class PyMetpy(PythonPackage):
     with weather data."""
 
     homepage = "https://github.com/Unidata/MetPy"
-    pypi     = "MetPy/MetPy-1.0.tar.gz"
+    pypi     = "MetPy/MetPy-1.0.1.tar.gz"
+    maintainers = ['dopplershift']
 
-    version('1.0', sha256='11b043aaa4e3d35db319e96bb9967eb9f73da653e155bca2d62f838108b100dc')
+    version('1.0', sha256='11b043aaa4e3d35db319e96bb9967eb9f73da653e155bca2d62f838108b100dc',
+            deprecated=True)
+    version('1.0.1', sha256='16fa9806facc24f31f454b898741ec5639a72ba9d4ff8a19ad0e94629d93cb95')
```

Comment:
Can you sort from newest to oldest? This is just a style thing, it doesn't affect the package.

Reply:
Done.